build.gradle

```diff
@@ -54,6 +55,18 @@ subprojects {
         }
     }
 
+    test {
```

Comment:
Just double checking, did you test to see if the report is generated properly per module?

Also, did you check if an aggregate report is generated at the root? For Codacy to work we will need that (or at least that is how it used to be).